.github/workflows/ossar-analysis.yml

```diff
@@ -1,20 +1,33 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
 # This workflow integrates a collection of open source static analysis tools
 # with GitHub code scanning. For documentation, or to provide feedback, visit
 # https://github.com/github/ossar-action
 name: OSSAR
 
 on:
   push:
+    branches: [ $default-branch, $protected-branches ]
   pull_request:
+    branches:
+
+permissions:
+  contents: read
 
 jobs:
   OSSAR-Scan:
     # OSSAR runs on windows-latest.
     # ubuntu-latest and macos-latest support coming soon
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
     runs-on: windows-latest
```

Comment:
I also noticed that `ossar-action` repository has sample file for `ubuntu-latest` - https://github.com/github/ossar-action/blob/main/.github/workflows/sample-workflow-ubuntu-latest.yml

I tried to replace `window-latest` with `ubuntu-latest` in our config, and it runs much faster - [2m 11s](https://github.com/FlowCrypt/flowcrypt-browser/actions/runs/9174491542/job/25225465955) on `windows` vs [23s](https://github.com/FlowCrypt/flowcrypt-browser/actions/runs/9174852536/job/25226581446) on `ubuntu`.

So let's switch to `ubuntu-latest` here

Reply:
Thanks Roma - I will check that and switch to the `ubuntu-latest` as the target machine.